### PR TITLE
Added a fn to ensure we have clean, attachable filenames no matter

### PR DESCRIPTION
**Pull request by danielglauser**
what the user's name is. Fixes [this bug](https://www.pivotaltracker.com/story/show/115509277) and will soon have a sister PR in [`voting-by-mail-works`](https://github.com/democracyworks/vote-by-mail-works/pull/4).

### CODE REVIEW

File: src/voter_registration_works/reg_form.clj [hunk 1]
```diff
@@ -71,13 +71,29 @@
               [ky val])))
         identity mp))
 
+(defn str->safe-for-email-attachments
+  "Given a String replaces all accented charaters with their
```
Comment (7maples) [inline src/voter_registration_works/reg_form.clj]: 'characters' is misspelled. I suggest one sentence capturing everything it does ("this, this and that") vs the "Also..." afterthought. Same for related PR with this same docstring.

**Comment by danielglauser** [comment]
@7maples once you approve of this new docstring I'll move it over to the other repo.

**Comment by 7maples** [comment]
Did you push it up? I don't see the change

**Comment by danielglauser** [comment]
@7maples Well...now I did!

**Comment by 7maples** [comment]
LGTM!

**Pull request merged**